src/background/services/secrets/SecretsService.ts

```diff
@@ -257,7 +282,8 @@ export class SecretsService {
   }
 
   async deleteImportedWallets(
-    ids: string[]
+    ids: string[],
+    walletConnectService: WalletConnectService
```

Comment:
Can we inject `WalletConnectService` in the constructor instead of passing it through the method parameters here?